lib/extensions/ar_miq_set.rb

```diff
@@ -35,44 +37,59 @@ def sets
   def make_memberof(set)
     set.add_member(self)
   end
+
+  def make_not_memberof(set)
+    set.remove_member(self)
+  end
 end # module ActsAsMiqSetMember
 
+module ActiveRecord
+  class Base
+    def self.acts_as_miq_set_member
+      include ActsAsMiqSetMember
+    end
+
+    def self.acts_as_miq_set(model_class = nil)
+      include ActsAsMiqSet
+
+      self.model_class = model_class unless model_class.nil?
```

Comment:
future note: if we want to support STI here, we probably need to use base_class or something